### PR TITLE
fix: Add conditional check for existing sops-age secret in bootstrap task

### PR DESCRIPTION
**Pull request by Deenyoro**
Add conditional check for existing sops-age secret in bootstrap task

### CODE REVIEW

**Comment by Deenyoro** [comment]
I know the main task file checks if sops is in the user directory here: 

```      - cmd: |
          if test -f ~/.config/sops/age/keys.txt; then
            echo WARNING: SOPS Age key found in home directory, this may cause conflicts.```
            
But I thought this change might help people that are defining their own Age key.

**Comment by onedr0p** [comment]
Overall I think this makes sense, I just want to get to my laptop to review it 🙂

File: README.md [hunk 1]
```diff
@@ -288,7 +288,7 @@ By default Flux will periodically check your git repository for changes. In orde
     https://flux-webhook.${bootstrap_cloudflare.domain}/hook/12ebd1e363c641dc3c2e430ecf3cee2b3c7a5ac9e1234506f6f5f3ce1230e123
     ```
 
-3. Navigate to the settings of your repository on Github, under "Settings/Webhooks" press the "Add webhook" button. Fill in the webhook url and your `bootstrap_github_webhook_token` secret and save.
+3. Navigate to the settings of your repository on Github, under "Settings/Webhooks" press the "Add webhook" button. Fill in the webhook URL and your bootstrap_github_webhook_token secret in config.yaml, Content type: application/json, Events: Choose Just the push event, and save.
```
Comment (onedr0p) [inline README.md:291]:
```suggestion
3. Navigate to the settings of your repository on Github, under "Settings/Webhooks" press the "Add webhook" button. Fill in the webhook URL and your `bootstrap_github_webhook_token` secret in config.yaml, Content type: application/json, Events: Choose Just the push event, and save.
```

File: README.md [hunk 2]
```diff
@@ -288,7 +288,7 @@ By default Flux will periodically check your git repository for changes. In orde
     https://flux-webhook.${bootstrap_cloudflare.domain}/hook/12ebd1e363c641dc3c2e430ecf3cee2b3c7a5ac9e1234506f6f5f3ce1230e123
     ```
 
-3. Navigate to the settings of your repository on Github, under "Settings/Webhooks" press the "Add webhook" button. Fill in the webhook url and your `bootstrap_github_webhook_token` secret and save.
+3. Navigate to the settings of your repository on Github, under "Settings/Webhooks" press the "Add webhook" button. Fill in the webhook URL and your `bootstrap_github_webhook_token` secret in config.yaml, Content type: application/json, Events: Choose Just the push event, and save.
```
Comment (onedr0p) [inline README.md:291]:
```suggestion
3. Navigate to the settings of your repository on Github, under "Settings/Webhooks" press the "Add webhook" button. Fill in the webhook URL and your `bootstrap_github_webhook_token` secret in `config.yaml`, Content type: `application/json`, Events: Choose Just the push event, and save.
```

**Review by onedr0p — APPROVED** [review]

**Pull request merged**